src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1178,6 +1223,7 @@ let daemon logger =
          Block_time.Controller.disable_setting_offset () ;
          let%bind coda = setup_daemon () in
          let%bind () = Mina_lib.start coda in
+         let () = Uptime_service.start coda in
```

Comment:
Can this go in `Mina_lib.start` with our other init code?

Reply:
The uptime service checks a mutable field in `Mina_lib.t`, so `Uptime_service` depends on `Mina_lib`.

To start the uptime service from `Mina_lib`, that library becomes dependent on `Uptime_service`, and we have a cycle.

We could change the `mutable` to a `ref`, and pass that, but that's kind of unpleasant.

Reply:
Could also pass an accessor function like `~get_foo:(fun () -> mina_lib.foo)`?

Reply:
Done (needed 3 accessors for mutable data, and pass several other bits from `Mina_lib.t`)